src/index.html

```diff
@@ -1,13 +1,353 @@
+              <a
+              href="tel:+1 234 5555-55-55"
+              class="icon icon--phone header__menu-call">
+              </a>
+
+              <a
+              href="#menu"
+              class="icon icon--menu header__menu-opener">
+              </a>
+            </div>
+          </div>
+
+          <h1 class="header__title">
+            Take the Streets
+          </h1>
+        </div>
+      </div>
+    </header>
+
+    <nav class="page__menu menu" id="menu">
+      <div class="container">
+        <div class="menu__content">
+          <div class="menu__top top-actions">
+            <a href="/" class="logo">
+              <img
+              class="logo__image"
+              src="./images/MyBiKE.svg"
+              alt="MyBiKE">
+            </a>
+
+            <a href="#"
+            class="icon icon__menu--crose">
+            </a>
+          </div>
+          <ul class="menu__list">
+            <li class="menu__item">
+              <a href="#" class="menu__link">Home</a>
+            </li>
+            <li class="menu__item">
+              <a href="#about-us" class="menu__link">about us</a>
+            </li>
+            <li class="menu__item">
+              <a href="#compare" class="menu__link">Compare Bikes</a>
+            </li>
+            <li class="menu__item">
+              <a href="#details" class="menu__link">Details</a>
+            </li>
+            <li class="menu__item">
+              <a href="#contacts" class="menu__link">Contacts</a>
+            </li>
+          </ul>
+
+          <a
+          href="tel:+1 234 5555-55-55"
+          class="menu__contacts menu__contacts--phone"
+          >
+            +1 234 5555-55-55
+          </a>
+          <a
+          href="tel:+1 234 5555-55-55"
+          class="menu__contacts menu__contacts--action"
+          >
+            Book a test ride
+          </a>
+        </div>
+      </div>
+    </nav>
+    <main>
+      <section class="page__section about" id="about-us">
+        <div class="container">
+          <div class="grid grid--desktop grid--tablet">
+            <div class="grid__item grid__item1-5-d grid__item1-3-t">
+              <h2 class="about__title">
+                Move Free
+              </h2>
+            </div>
+
+            <div class="grid__item grid__item7-12-d grid__item4-6-t">
+              <p class="about__parahraf">
+              Shift your ride, not gears. Find the fastest way to move in the city as the bike adapts intuitively to power the speed you need.
+              </p>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="page__section compares" id="compare">
+        <div class="container">
+          <h2 class="page__section-title">
+            Compare Bikes
+          </h2>
+          <div class="grid grid--desktop grid--tablet">
+            <div class="compares__item grid__item grid__item1-4-d grid__item2-5-t">
+              <div class="compare">
```

Comment:
Bad naming! It's easy to get these classes mixed up.

Reply:
fixed.